src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.